### ISSUE TITLE
Explicitly state RCB requires Kodi Jarvis or above

### ISSUE THREAD

**Issue by bruny** [maloep/romcollectionbrowser#247]
I'm running Isengard on Mac and RCB from source fails on startup with the following error:

```
22:42:57 T:123145320087552  NOTICE: Unhandled exception occured during execution of RCB:
22:42:57 T:123145320087552  NOTICE: __init__() got an unexpected keyword argument 'encoding'
```

This appears to be due to the following line in config.py:

```
tree = ElementTree()
parser = XMLParser(encoding='utf-8') 
tree.parse(self.configFile, parser)
```

As far as I can tell, this looks like the issue is with XMLParser in Python 2.6 (which is shipped with Kodi Isengard on Mac).

The easy fix is to upgrade to Jarvis, which has been at release for over 6 months, and runs Python 2.7 which fixes the above error. But it would be worth explicitly tying RCB to a release pattern, e.g. most recent, current alpha minus 1. That way we can remove some of the legacy code (e.g. there's references to Dharma and Eden features in the codebase) as well.

(Ref http://forum.kodi.tv/showthread.php?tid=235078&pid=2079543#pid2079543)

**Comment by bruny** [comment]
One other reason is that xbmcvfs.exists is not guaranteed to work for directories, since it caches the result (ref https://github.com/xbmc/xbmc/pull/8531) - but this wasn't fixed until Krypton alphas.

**Comment by bruny** [comment]
Upgrading to Jarvis doesn't fix this issue, since Python 2.6 is still shipped (at least on OSX).

Will have to put in a workaround for this if we are to support current version minus one.

**Comment by bruny** [comment]
Created another PR #263 to add a check for which Python version is in use, which should fix the crash reported recently in the forums for this problem

**Comment by bruny** [comment]
Closed by #263 and #254
